draft-ietf-gnap-core-protocol.md

```diff
@@ -204,29 +194,37 @@ role of the AS as defined by the protocol.
 In addition to the roles above, the protocol also involves several 
 elements that are acted upon by the roles throughout the process.
 
+Attribute
+: characteristics related to an end-user.
+
 Access Token
-: A credential representing a set of access rights
-delegated to the client instance. The access token is created by the AS, consumed
-and verified by the RS, and issued to and carried by the client instance. The contents
-and format of the access token are opaque to the client.
+: digitally signed data that contains specific rights and/or attributes.
+
+    Note: an access token can be first issued to an end-user (usually requiring his authentication) and subsequently refreshed.
```

Comment:
Tokens aren't issued to people, they're issued to software -- also the spec currently uses "rotate" instead of "refresh" language, though that might not be wholly consistent yet so we should check that.

```suggestion
    Note: an access token can be first issued to an client instance (requiring authorization by the RO) and subsequently rotated.
```

Reply:
same remark as before. You're right, but that issue shows we need to explain the relationship between the user and the software (aka delegated access).